.gitignore

```diff
@@ -1,8 +1,22 @@
 *.iml
 .gradle
 /local.properties
-/.idea/workspace.xml
-/.idea/libraries
+.idea/workspace.xml
```

Comment:
Took from https://www.gitignore.io, using patterns `Android` and `AndroidStudio`.  Thanks.